.github/workflows/update-hacl-packages.yml

```diff
@@ -0,0 +1,43 @@
+name: Update hacl-packages
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  update-hacl-packages:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout hacl-star
+        uses: actions/checkout@v3
+        with:
+          path: hacl-star
+      - name: checkout hacl-packages
+        uses: actions/checkout@v3
+        with:
+          repository: cryspen/hacl-packages
+          path: hacl-packages
+          token: ${{ secrets.HACL_BOT }}
+      - name: update hacl-packages
+        working-directory: hacl-packages
+        run: |
+          rm -r src/* include/*
+          cp ../hacl-star/dist/gcc-compatible/*.c src/
+          cp ../hacl-star/dist/gcc-compatible/*.h include/
+          cp -r ../hacl-star/dist/gcc-compatible/internal include/
+      - name: push to hacl-packages
+        working-directory: hacl-packages
+        run: |
+          branch=hacl-star-${{ github.head_ref }}
+          token=${{ secrets.HACL_BOT }}
+          git config --local user.name "Hacl Bot"
+          git config --local user.email "hacl-star@mailo.com"
+          git checkout -b $branch
+          git add src include
+          git commit -m "[CI] update code"
+          git push --force --set-upstream  origin $branch
+      - name: comment pr
+        uses: thollander/actions-comment-pull-request@v1
+        with:
+          message: "[CI] corresponding branch on hacl-packages: https://github.com/cryspen/hacl-packages/tree/hacl-star-${{ github.head_ref }}"
```

Comment:
Let's link to the action triggered on this branch. That's what people should look at.
I'd be fine doing this in a followup. Nothing is triggered yet anyway.

Reply:
I removed it for now